### DISCUSSION TITLE
DB Import fails for tables with Generated Column

### DISCUSSION THREAD

**Discussion by eyounelson** [spatie/laravel-backup#1701] [Ideas]
I have a table with a generated column that was created via Laravel migration like so:

```php
$table->string('pair')->after('rate')->virtualAs('concat(source, \' to \', destination)');
```

After the DB has been backed up, attempting to restore such a table fails because the DB export attempts to insert into the `pair` column which it shouldn't.

I'm wandering whether this is a MySQL issue or something the package can properly handle? Right now I have to edit my DB dumps and update such insert statements.

**Comment by freekmurze** [comment]
I'm open for PRs that would handle this!
